docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -52,12 +52,57 @@ For customers with applications using right to left directionality, the example
 ```
 Also see: [Internationalization](react/getting-started/internationalization)
 
+
 #### 3. Legacy Authenticator exports removed
 
 If you are still importing the version 1.x of the Authenticator using the `/legacy/` path, please update to the current version. Exports removed include:
 
 `AmplifyAuthenticator`, `AmplifySignIn`, `AmplifySignOut`, `withAuthenticator`
 
+#### 4. Update the handleSignUp function override call
```

Comment:
As mentioned above in angular migration guide, 4 and 5 should be under the same heading, since they require each other.